app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+	 def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(comment_params)
+    redirect_to post_path(@post)
+  end
```

Comment:
Again, Bad allignment

Reply:
oops sorry its my mistake.Ok will correct it and resend to u